app/code/Magento/Customer/Observer/AfterAddressSaveObserver.php

```diff
@@ -123,7 +141,14 @@ public function execute(\Magento\Framework\Event\Observer $observer)
         /** @var $customerAddress Address */
         $customerAddress = $observer->getCustomerAddress();
         $customer = $customerAddress->getCustomer();
-
+        
+        if ($customer->getId()) {
+            $currentTimestamp = $this->date->gmtDate();
+            $customerModel = $this->_customerFactory->create()->load($customer->getId());
+            $customerModel->setUpdatedAt($currentTimestamp);
+            $customerModel->save();
```

Comment:
We can't save the customer from `AfterAddressSaveObserver` because the customer save method (`Magento/Customer/Model/ResourceModel/CustomerRepository.php` , `266`) calls save the address method, and there will be recursion.

Reply:
Ok! Let me make some change!